src/MSBuild/app.amd64.config

```diff
@@ -215,8 +215,8 @@
         </dependentAssembly>
         <dependentAssembly>
           <assemblyIdentity name="System.Runtime.CompilerServices.Unsafe" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-          <bindingRedirect oldVersion="0.0.0.0-6.0.0.0" newVersion="6.0.0.0" />
-          <codeBase version="6.0.0.0" href="..\System.Runtime.CompilerServices.Unsafe.dll"/>
+          <bindingRedirect oldVersion="0.0.0.0-6.0.1.0" newVersion="6.0.1.0" />
```

Comment:
6.0.1.0 is the value that was wrong in the latest published Unsafe package. We just fixed it to be 6.0.2.0, so we will have to update this after we ship the new version to nuget.org.

https://github.com/dotnet/maintenance-packages/blob/f62ca0009b038cab4725a720f386623a969d73ad/src/System.Runtime.CompilerServices.Unsafe/Versioning.props#L7